packages/dcos-metrics/extra/dcos-metrics-master.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Metrics Service: DC/OS Metrics Service for Application, Container and Host Level Metrics.
```

Comment:
This description should be different between master / agent so people can tell which they're looking at in the ui